package-lock.json

```diff
@@ -6135,8 +6150,9 @@
       "dependencies": {
         "broccoli-funnel": {
           "version": "1.0.2",
-          "resolved": "https://registry.npmjs.org/broccoli-funnel/-/broccoli-funnel-1.0.2.tgz",
+          "resolved": "https://artifactory.ciena.com/api/npm/blueplanet-npm/broccoli-funnel/-/broccoli-funnel-1.0.2.tgz",
```

Comment:
`https://registry.npmjs.org/...` entries should not being changed to `https://artifactory.ciena.com/api/npm/blueplanet-npm...` ones